spec/models/theme_spec.rb

```diff
@@ -1347,6 +1347,62 @@ def included_settings(id)
         "deletions" => [{ "key" => "setting_that_will_be_removed", "val" => 1023 }],
       )
     end
+
+    it "does not raise an error and does not create `ThemeSettingsMigration` record for out of sequence migration when `allow_out_of_sequence_migration` kwarg is set to true" do
+      migration_field.update!(value: <<~JS)
+        export default function migrate(settings) {
+          settings.set("integer_setting", 2);
+          return settings;
+        }
+      JS
```

Comment:
Same question here, not really why we have this and not just use expect to change later